original/java/db.properties

```diff
@@ -0,0 +1,5 @@
+#mysql DB properties
```

Comment:
better practice to have this as e.g. `db.properties.default` that people will copy to `db.properties` to run the code. this way there will be no code conflicts as people use the code